app/tools/android/make_apk.py

```diff
@@ -314,12 +314,13 @@ def CleanNativeLibrary(library_path, arch):
 
 
 def CopyCompressedLibrary(native_path, library_path, raw_path, arch):
-  # need dummy library file to keep the arch info if enable lzma.
+  # copy dummy library file to keep the arch info if it's available.
   arch_path = os.path.join(library_path, arch)
-  if (not os.path.isdir(arch_path)):
-      os.mkdir(arch_path)
   dummy_library = os.path.join(native_path, DUMMY_LIBRARY);
-  shutil.copy(dummy_library, arch_path)
+  if (os.path.isfile(dummy_library)):
```

Comment:
Yes, this is google python coding style, please check https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Parentheses#Parentheses